glusterd2/snapshot/store-utils.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"strings"
```

Comment:
this import should be changed as below to adhere coding guidelines [guidelines](https://github.com/gluster/glusterd2/blob/master/doc/coding.md)
```
      "context"
      "encoding/json"
      "errors"
      "strings"

       gdstore "github.com/gluster/glusterd2/glusterd2/store"
       "github.com/gluster/glusterd2/glusterd2/volume"

       "github.com/coreos/etcd/clientv3"
       log "github.com/sirupsen/logrus"
```